### PR TITLE
GitHub actions workflows

### PR DESCRIPTION
**Pull request by stefanotorresi**
This PR removes Travis as a CI system and introduces GitHub actions via two separated worfklows for the exporter and the dashboards.

The following GitHub secrets need to be configured to make this work:
- `OBS_DOWNSTREAM_PROJECT`
- `OBS_PASS`
- `OBS_PROJECT`
- `OBS_USER`

### CODE REVIEW

File: .github/workflows/dashboards-ci.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    - uses: actions/setup-node@v1
+    - run: npm -g install jsonlint
+    - name: validate JSON
+      run: find dashboards -name "*.json" -type f -exec jsonlint -c {} \;
+
+  obs-commit:
+    needs: build
+    if: github.ref == 'refs/heads/master'
+    runs-on: ubuntu-latest
+    container: shap/continuous_deliver
```
Comment (MalloZup) [inline .github/workflows/dashboards-ci.yml:31]: is this a container we have ? maybe we need to update confluence page regarding this

File: .github/workflows/exporter-ci.yml [hunk 2]
```diff
@@ -0,0 +1,103 @@
+name: Exporter CI
```
Comment (MalloZup) [inline .github/workflows/exporter-ci.yml:1]: for all workflow, I would like to have an higher lever description to see what is the intent of the workflow

like. Build and test XX foo .

File: .github/workflows/dashboards-ci.yml [hunk 3]
```diff
@@ -0,0 +1,49 @@
+name: Dashboards CI
```
Comment (MalloZup) [inline .github/workflows/dashboards-ci.yml:1]: same here, highlevel rationale is alwasy appreciate as comment

check json on pr, build only on master

File: .github/workflows/dashboards-ci.yml (hunk 1 above)
Comment (stefanotorresi) [reply .github/workflows/dashboards-ci.yml:31]: yeah this is hosted on docker hub and sourced from https://github.com/arbulu89/continuous-delivery

File: .github/workflows/exporter-ci.yml (hunk 2 above)
Comment (stefanotorresi) [reply .github/workflows/exporter-ci.yml:1]: I will add some comments with a more exhaustive description: I can't use the `name` field, that's meant to be a short name.

Comment (MalloZup) [reply .github/workflows/exporter-ci.yml:1]: yeah just add comment

File: .github/workflows/dashboards-ci.yml (hunk 1 above)
Comment (MalloZup) [reply .github/workflows/dashboards-ci.yml:31]: ok I think this was an `howto` tutorial. I will move it to `CI` session. I think personally the howto section is somehow to much generic like `utils` directory without any context

Comment (MalloZup) [reply .github/workflows/dashboards-ci.yml:31]: @stefanotorresi  i'm moving the page on the CI section. feel free to add it maybe with github action context too

Comment (stefanotorresi) [reply .github/workflows/dashboards-ci.yml:31]: yeah, good call!

**Pull request merged**